lib/spack/spack/cmd/find.py

```diff
@@ -109,6 +109,10 @@ def setup_parser(subparser):
     subparser.add_argument(
         '--end-date', help='latest date of installation [YYYY-MM-DD]'
     )
+    subparser.add_argument(
+        '-b', '--bootstrap', action='store_true',
```

Comment:
Do we want a short version for this argument, or should it be only `--bootstrap`?

Also, have you thought about what happens if the user manually sets their store to be the same as the Spack bootstrap store?

Reply:
> Do we want a short version for this argument, or should it be only --bootstrap?

No preference on this one. Let me know if you want it to change.

> Also, have you thought about what happens if the user manually sets their store to be the same as the Spack bootstrap store?

No, I didn't account for that case at all. My assumption is that we'll put the store in a directory managed by Spack in a path that is not "easily" selected accidentally.

Reply:
Can we just add a check when we read in the configs that will break if the user tries to set their install_tree to be the same as the Spack one? It shouldn't happen by accident, but if someone tries to get too clever about it we can stop them.

Reply:
Should be done in 4129912